.github/workflows/stale.yml

```diff
@@ -33,8 +33,8 @@ jobs:
             However, unless the issue is a concrete proposal that can be implemented, we recommend starting a language discussion on the [forum](https://forum.soliditylang.org) instead.
           ascending: true
           stale-issue-label: stale
-          close-issue-label: closed-due-inactivity
-          exempt-issue-labels: 'bug :bug:,epic,roadmap,selected-for-development,must have,must have eventually,SMT'
```

Comment:
To be honest I kind of prefer having labels without spaces. But they would not look nice with emojis. So I'm fine either way.

Reply:
If everyone wants dashes, we could switch, but it's just not what I tried to do with this PR. We've been already using the convention with spaces for quite a while, and I only wanted to quickly fix the new labels that do not follow it.